lighty-examples/lighty-controller-springboot-netconf/pom.xml

```diff
@@ -80,6 +80,11 @@
             <artifactId>jcasbin</artifactId>
             <version>1.79.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents.client5</groupId>
+            <artifactId>httpclient5</artifactId>
+            <version>5.4.1</version>
```

Comment:
why? :)

now, we need to manage this version, for example, spring 3.4.3 has increased this version

Reply:
https://github.com/PANTHEONtech/lighty/actions/runs/13975457568/job/39128000126?pr=2168#step:9:3067 
The reason can be seen here. Spring needs this dependency. If its not present, it fails to find it.

Reply:
Please, see: https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.4-Release-Notes